src/commonMain/kotlin/exchange.dydx.abacus/utils/Map+Utils.kt

```diff
@@ -141,3 +141,15 @@ fun IMap<String, String>.toUrlParams(): String =
     entries.joinToString("&") {
         it.key + "=" + it.value
     }
+
+fun Map<String, Any>.toCamelCaseKeys(): Map<String, Any> {
```

Comment:
if people feel strongly i can write a quick test for this method. i just didn't feel like it was urgently necessary since we test that it works in the `SkipRoutePayloadProcessorTests`